src/events/balancer.py

```diff
@@ -76,6 +80,7 @@ def _balancer_layer(self) -> Layer:
         extra_opts = [
             # FIXME: Port already in use by the broker. To be fixed once we have CC_JMX_OPTS
             # f"-javaagent:{CharmedKafkaPaths(BROKER).jmx_prometheus_javaagent}={JMX_EXPORTER_PORT}:{CharmedKafkaPaths(BROKER).jmx_prometheus_config}",
+            f"-javaagent:{CharmedKafkaPaths(BROKER).jmx_prometheus_javaagent}={JMX_CC_PORT}:{self.workload.paths.jmx_cc_config}",
```

Comment:
Do you know why the prometheus jar is absent from `/opt/cruise-control`?

Reply:
It is the same for both, so we don't need it anywhere else. Maybe it would be better to move it somewhere common and to have that extra "common" place be reflected on the charm.

Reply:
Seems like something to discuss next grooming session if we want to make a ticket out of it. I'll make a note